recipes/configs/qwen2/0.5B_full_low_memory.yaml

```diff
@@ -0,0 +1,77 @@
+# Config for single device full finetuning in full_finetune_single_device.py
```

Comment:
One thing that's a bit unclear (maybe this is our fault for how we've named our existing recipes), but configs like `*_low_memory` tend to use techniques that may be slower for the sake of being runnable on more constrained hardware. E.g. `llama2/7B_full_low_memory` should run in under 16GB of VRAM but won't be super fast. So I would question whether we actually need to apply these techniques for these smaller models: it may be better to use non-paged optimizer and disable optimizer in backward. (We could probably even get away with disabling activation checkpointing for these smaller models, but maybe that would bite us at longer sequence lengths)

Reply:
I would say we delete these recipes entirely no? Or rename to `full_single_device` to denote that it doesn't have low memory optimizations?

Reply:
Hi, I have renamed all `qwen*_full_low_memory` recipes to `qwen*_full_single_device`.